docs/user/dql/aggregations.rst

```diff
@@ -182,7 +182,7 @@ AVG
 Description
 >>>>>>>>>>>
 
-Usage: AVG(expr). Returns the average value of expr.
+Usage: AVG(expr). Returns the average value of `expr`. `expr` could be of any of the numeric and datetime data types.
```

Comment:
...can be double or datetime data types, right?

Reply:
Any numeric or any datetime type. Should I list them?
Please, correct my wording if needed.